src/test/java/org/dinosaur/foodbowl/domain/comment/application/CommentServiceTest.java

```diff
@@ -55,7 +69,77 @@ void saveFailWithWrongMemberId() {
 
             assertThatThrownBy(() -> commentService.save(Long.MAX_VALUE, commentCreateRequest))
                     .isInstanceOf(FoodbowlException.class)
-                    .hasMessage("등록되지 않은 회원입니다.");
+                    .hasMessage(MEMBER_NOT_FOUND.getMessage());
+        }
+    }
+
+    @Nested
+    @DisplayName("댓글을 수정할 때 ")
+    class UpdateComment {
+
+        @Test
+        @DisplayName("정상적으로 내용이 수정된다.")
+        void updateComment() {
+            Member member = memberTestSupport.memberBuilder().build();
+            Post post = postTestSupport.postBuilder().build();
+            Comment comment = commentTestSupport.builder().member(member).post(post).message("돈까스 드시죠").build();
+            CommentUpdateRequest commentUpdateRequest = new CommentUpdateRequest("아, 돈까스 말고 그냥 국밥 드시죠");
+            em.flush();
+            em.clear();
+
+            commentService.updateComment(comment.getId(), member.getId(), commentUpdateRequest);
+            em.flush();
+            em.clear();
+            Comment updatedComment = commentRepository.findById(comment.getId()).get();
+
+            assertThat(updatedComment.getMessage()).isEqualTo(commentUpdateRequest.getMessage());
```

Comment:
```suggestion
            Member member = memberTestSupport.memberBuilder().build();
            Post post = postTestSupport.postBuilder().build();
            Comment comment = commentTestSupport.builder().member(member).post(post).message("돈까스 드시죠").build();
            CommentUpdateRequest commentUpdateRequest = new CommentUpdateRequest("아, 돈까스 말고 그냥 국밥 드시죠");
            
            em.flush();
            em.clear();

            commentService.updateComment(comment.getId(), member.getId(), commentUpdateRequest);
            
            em.flush();
            em.clear();
            
            Comment updatedComment = commentRepository.findById(comment.getId()).get();
            assertThat(updatedComment.getMessage()).isEqualTo(commentUpdateRequest.getMessage());
```
개행으로 조금 더 시나리오를 구분해보는건 어떨까요?

Reply:
given, when, then 구조를 가져가려고 3문단에 나눠서 작성했는데
영속성 컨텍스트를 비우고 변경된 댓글을 가져오는 과정까지 when 절로 볼 수 있지 않을까요?!

Reply:
그런 의도를 가지고 있었다면 좋은 것 같습니다!